code/modules/cargo/console.dm

```diff
@@ -24,6 +24,11 @@
 							"<span class='notice'>You adjust [src]'s routing, unlocking special supplies.</span>")
 		emagged = TRUE
 
+		// Emagging also unlocks the Broad-banded supplies
+		var/obj/item/weapon/circuitboard/cargo/board = circuit
+		circuit.contraband = TRUE

```

Comment:
needs to be board.contraband
also the thing should become permanently emagged and broad-banded